web/client/utils/GeoStoryUtils.js

```diff
@@ -454,7 +454,7 @@ export const parseHashUrlScrollUpdate = (url, hash = '', storyId, sectionId, col
             return replace(url, `${storyIds[1]}`, `${sectionId}`);
         }
         if (hash.includes('shared')) {
-            return `${url}${sectionId}`;
+            return `${url}/${sectionId}`;
```

Comment:
While testing other functionality I removed manually the section id from the url so I got the path with the trailing slash `#/geostory/:gid/` this caused a wrong parsing of the path stacking new id on the same path.

see the url in this gif
![trailing-slash-1](https://user-images.githubusercontent.com/19175505/88519762-e937b980-cff2-11ea-8970-7972de2fee27.gif)

I'm not sure the issue is related on this function anyway if the url argument is expected without trailing slash we should ensure is it removed from it in this parsing function before to add new parts to the path.